### PR TITLE
COL-766 Feature/site font review

### PR DESCRIPTION
**Pull request by katybeck**
* Removing hosted fonts
* adding in google fonts call
* evening out type sizes and weights across site
* building /about/stylesheet where you can check basic sizes, header tags, paragraph tags, etc.

### CODE REVIEW

**Comment by QuinnLee** [comment]
Looks good, I added the content security policy for gfonts.  
http://ci-deploys.cid-labs.com/cid-harvard/atlas-subnational-frontend/feature/site-font-review/257/

File: Brocfile.js [hunk 1]
```diff
@@ -83,27 +83,6 @@ app.import('bower_components/font-awesome/fonts/FontAwesome.otf', { destDir: 'fo
 app.import('bower_components/cidcons/css/cidcons-codes.css');
 app.import('bower_components/cidcons/font/cidcons.woff', { destDir: 'fonts' });
 
-// Platform
```
Comment (QuinnLee) [inline Brocfile.js]: Can we remove these files from the `vendor` dir?

Comment (katybeck) [inline Brocfile.js]: Totally forgot to do that part - yes we can remove them.

Comment (QuinnLee) [inline Brocfile.js]: Done!

**Comment by QuinnLee** [comment]
QA can you look at this in FF and IE?

**Comment by makmanalp** [comment]
On Firefox, build 259 of this branch, homepage screenshots:

screen shot 2015-12-07 at 14 23 49

Vs current datlas-beta:
screen shot 2015-12-07 at 14 22 44

And fonts page:
screen shot 2015-12-07 at 14 25 07

Looks good to me! Don't have IE.

**Comment by makmanalp** [comment]
Safari also looks good but might need proper review, not quite sure what to look for:

screen shot 2015-12-07 at 2 26 12 pm

**Comment by makmanalp** [comment]
One thing is that the edges of the larger titles are super jaggedy, might be worth playing with text-rendering or font-smoothing in another ticket maybe?

**Pull request merged**